tpcx_bb/cluster_configuration/cluster-startup.sh

```diff
@@ -0,0 +1,60 @@
+#IB, NVLINK, or TCP
+CLUSTER_MODE=$1
+
+MAX_SYSTEM_MEMORY=$(free -m | awk '/^Mem:/{print $2}')M
+DEVICE_MEMORY_LIMIT="25GB"
+POOL_SIZE="30GB"
+
+# Fill in your environment name and conda path on each node
+CONDA_ENV_NAME="rapids-tpcxbb-20200706"
+CONDA_ENV_PATH="/raid/nicholasb/miniconda3/etc/profile.d/conda.sh"
+
+# TODO: Unify interface/IP setting/getting for cluster startup
+# and scheduler file
+INTERFACE="ib0"
+
+# TODO: Remove hard-coding of scheduler
+SCHEDULER="exp01"
```

Comment:
Can we  push the configs users are supposed to change above and together so that user changes everything at one go.

Reply:
Yes. Let's unify in a follow-up